src/components/TrailForm.js

```diff
@@ -169,6 +176,7 @@ class TrailForm extends React.Component {
 
 
               <HydrantList
+                trailHydrants={trailHydrants}
                 trail={trail}
                 hydrants={hydrants}
```

Comment:
Enjoy the fruits of your labors! Trails and hydrants are both unused props now, get rid of them.